lighthouse-core/test/audits/dobetterweb/link-blocking-first-paint-test.js

```diff
@@ -39,10 +39,16 @@ describe('Block First Paint audit', () => {
   });
 
   it('fails when there are links found which block first paint', () => {
+    const linkDetails = {
+      href: 'http://google.com/css/style.css',
+      disabled: false,
+      media: '',
+      rel: 'stylesheet'
+    };
     const auditResult = LinkBlockingFirstPaintAudit.audit({
       LinksBlockingFirstPaint: {
         items: [{
-          url: 'http://google.com/style.css',
+          link: linkDetails,
           transferSize: 100,
           spendTime: 100
         }],
```

Comment:
It wouldn't be testing much. The gatherer doesn't return links that are `.disabled == true`. We'll never see a link that's disabled in this test. Checking in the the gatherer test would also be contrived. That test should really use fixture data instead of constructing it's own stuff.